ginga/AutoCuts.py

```diff
@@ -628,6 +628,20 @@ def fitting(x, slope, intercept):
 
         return (float(locut), float(hicut))
 
+class Scaling(AutoCutsBase):
+
+    def __init__(self, logger):
+        super(Scaling, self).__init__(logger)
+        self.kind = 'scaling'
+
+    def calc_cut_levels(self, image):
+        # Get max and min of first slice
+        first = image.get_mddata()[0]
+        loval = numpy.nanmax(first)
+        hival = numpy.nanmin(first)
+
```

Comment:
`nanmin` and `nanmax` are reversed here.

But I don't think this is the correct implementation.  What this does is to apply the min/max values from the first slice to all slices.  Is that actually what is requested?